docs/_documentations/mdt-eclipse-usingadifferenttemplate.md

```diff
@@ -14,9 +14,39 @@ parent: usingcodewind
 
 When you create a new project, you can choose from the default set of templates available, or you can choose from a template source of your own. Use Codewind to manage templates. Templates can make application development easier by providing a structure and boilerplate code to help get a new project started.
 
-- To open the Template Source Manager, right-click **Projects (Local)**.
-- Click **Manage Template Sources** to add a source to the table.
-- You can perform the following actions in the dialog window:
-  - Use the check boxes to enable or disable sources.
-  - Use **Add** to add a new source.
-  - Use **Remove** to remove non-default sources.
\ No newline at end of file
+- To open the Template Source Manager, right-click **Projects (Local)**, and then click **Manage Template Sources**. The **Template Source Manager** appears. 
```

Comment:
This is now just **Local** instead of **Projects (Local)** (part of the hybrid changes)

Reply:
Corrected